CMakeLists.txt

```diff
@@ -273,11 +273,12 @@ else()
 endif()
 
 if(MINGW)
-  set_property(TARGET SDL2 APPEND_STRING PROPERTY LINK_FLAGS " -nostdlib") # libgcc is needed for 32 bit (x86) builds
-  if(USE_GCC)
-    target_link_libraries(SDL2 PRIVATE "gcc")
-    set_property(TARGET SDL2 APPEND_STRING PROPERTY LINK_FLAGS " -static-libgcc") # libgcc is needed for 32 bit (x86) builds
+  set_property(TARGET SDL2 APPEND_STRING PROPERTY LINK_FLAGS " -nostdlib")
+  if(USE_CLANG)
+    target_compile_options(SDL2 PRIVATE "-rtlib=libgcc")
```

Comment:
maybe add this only to LDFLAGS to avoid -Werror?

Reply:
I'm not sure. This is a compiler flag for linux clang.
Don't you need this flag during compilation such that the compiler knows what runtime library to use?